app/models/illiad_openurl.rb

```diff
@@ -0,0 +1,35 @@
+###
+#  Class to handle creation of ILLiad OpenURL request
+###
+class IlliadOpenurl
+  def initialize(scan, redirect_url)
+    @scan = scan
+    @redirect_url = redirect_url
+  end
+
+  def illiad_page
+    { 'Action': '10', 'Form': '30', 'rft.genre': 'scananddeliver' }
+  end
+
+  def illiad_item_data
+    { 'rft.location': @scan.origin,
+      'rft.call_number': @scan.holdings.first.callnumber,
+      'rft.item': @scan.holdings.first.barcode }
```

Comment:
Yes, `Scan#barcodes` is the array of selected barcodes (I think that plumbing may have not been there when this code was first introduced).

However; in this context, a request object where holdings have been selected, they are effectively the same. Holdings will be limited to only the user selected holdings, and since a scan can only have 1 item requested, it will be the user's selection. https://github.com/sul-dlss/sul-requests/blob/master/app/models/concerns/holdings.rb

This approach may end up causing issues down the road if we ever decide to expand the number of items a user may select for a scan since this is hard-coded to the first.